File: attic/pre2010/python/fizzbuzz.py

for n in xrange(1,100):
    if (n % 3 == 0 and n % 5 == 0 ):
        print "FizzBuzz"
    elif (n % 3 == 0):
        print "Fizz"
    elif ( n % 5 == 0 ):
        print "Buzz"
    else:
        print n

File: algorithms/lua/fizzbuzz.lua

for n=1,100 do
    if (n % 3 == 0 and n % 5 == 0 ) then
        io.write ( "FizzBuzz\n" )
    elseif (n % 3 == 0) then
        io.write ( "Fizz\n" )
    elseif ( n % 5 == 0 ) then
        io.write ( "Buzz\n" )
    else
        io.write ( n )
        io.write ( "\n")
    end
end
